go.hlb

```diff
@@ -4,25 +4,31 @@ export crossBinaries
 
 export lint
 
-fs build(fs src, string package) {
+string versionCmd() {
+	format "git describe --match 'v[0-9]*' --tags --dirty='.dirty' --always | sed 's/^v//'"
+}
+
+fs build(fs src, string package, string verPackage) {
 	image "golang:1.14-alpine"
 	run "apk add -U git gcc libc-dev"
 	env "GO111MODULE" "on"
 	dir "/go/src/hlb"
 	run string {
-		format "/usr/local/go/bin/go build -o /out/binary -ldflags '-linkmode external -extldflags -static' -a %s" package
```

Comment:
I think one feature we should implement is if the function has 0 arguments, you should be able to call it without a `string` func literal.

Reply:
Created design issue: #30